app/src/main/java/com/dmonzonis/nookpendium/FilterManager.kt

```diff
@@ -21,6 +22,10 @@ class FilterManager(private val activity: CollectionActivity) {
     }
 
     fun applyFilters(recordset: Recordset) {
+        // Apply search filter separately
+        // TODO: Search by price if the searchQuery is a number
```

Comment:
idk about this, seems weird, what we could have instead is a filter with ranges for prices (from 2000 up, 2000 to 5000, etc)

Reply:
I guess, it was just an idea